presets/4.5/other/multigp_pro_spec.txt

```diff
@@ -0,0 +1,215 @@
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: OTHER
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: 6S, race, 7 inch, 7", Shames, MGP, Sharjah, multigp
+#$ AUTHOR: Armando Gallegos (Mondo)
+
+#$ FORCE_OPTIONS_REVIEW: TRUE
+#$ PARSER: MARKED
+
+#$ DESCRIPTION:
+#$ DESCRIPTION: <img src="https://github.com/betaflight/firmware-presets/assets/2925027/a75803ff-d1f8-4a47-9a5f-1b5d57d28f56" width="90px" style="display: block; float: left; margin-right: 10px; margin-top: 20px"/>
+#$ DESCRIPTION: <h1>MultiGP PRO Spec</h1>
+#$ DESCRIPTION: Tune + RPM limit + extras.
+#$ DESCRIPTION:
+#$ DESCRIPTION:
+#$ DESCRIPTION: This tune is what was used by MultiGP in Sharjah stock race. Made for ~2807 motors 1300kv, 6S, HQ props 7x4x3, and heavy RPM limit.
+#$ DESCRIPTION:
+#$ DESCRIPTION: More information about [MultiGP PRO Spec](https://www.multigp.com/prospec/)
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: This preset requires Betaflight 4.5 flashed with RACE_PRO option, or Betaflight special release KAACK 4.5.
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Miscellaneous option
+#$ DESCRIPTION: is checked by default, and includes several usefull settings:
+#$ DESCRIPTION: - Accelerometer and barometer OFF
+#$ DESCRIPTION: - Motor KV = 1300 (change in CLI if needed)
+#$ DESCRIPTION: - RPM limit ON
+#$ DESCRIPTION: - Small angle 180 (arming at any angle)
+#$ DESCRIPTION: - Props out (yaw_motors_reversed = ON)
+#$ DESCRIPTION: - Features: LED strip and OSD
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Things to note:
+#$ DESCRIPTION: - **YOU HAVE TO USE RPM FILTERING WITH THIS TUNE!** Failure to do so might result in fire 🔥
+#$ DESCRIPTION: - This preset is setting up DShot600: if your setup has errors in the motor tab using bidirectional DShot, change to **8k/4k and DShot300**
+#$ DESCRIPTION: - To test arm the quad with props on, it should **sound clean with no grinding**. If it passes that then hover test and check motor temps.
+#$ DESCRIPTION: - **Safety first! If anything is off, don't fly it!!!**
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Second note... Radio links:
+#$ DESCRIPTION: 1. Make sure your radio firmware is up to date using either EdgeTX or OpenTX
+#$ DESCRIPTION: 2. Make sure your **ADC Filter is OFF** in the hardware page
+#$ DESCRIPTION: 3. Go to the radio (RC_LINK) presets and apply the correct setup for your system and link speed
+#$ DESCRIPTION:
+
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/475
+
+#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
+#$ INCLUDE: presets/4.5/tune/defaults.txt
+#$ INCLUDE: presets/4.5/filters/defaults.txt
+
+# tune/filters
+
+set dshot_bidir = ON
+set motor_pwm_protocol = DSHOT600
+set dyn_notch_count = 2
+set dyn_notch_max_hz = 550
+set simplified_gyro_filter_multiplier = 60
+set rpm_filter_weights = 100,50,100
+
+set anti_gravity_gain = 90
+set iterm_relax_cutoff = 10
```

Comment:
this is really low. Have you tried the quad with more? 30-45?

Reply:
Will be about 30ms latency on I-term accumulation, not too low for a big 1100g drone?

But I will revisit the tune once I get my hands on it.